### PR TITLE
move getAddress method from two servlets to util file

### PR DESCRIPTION
**Pull request by Parachuteuk**
**What does this PR do?**
move getAddress method from two servlets to util file
**Why are these changes required?**

**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

File: framework/src/main/java/org/tron/core/services/http/Util.java [hunk 1]
```diff
@@ -439,4 +441,27 @@ public static void printAccount(Account reply, HttpServletResponse response, Boo
...
+    String addressParam = "address";
+    String addressStr = request.getParameter(addressParam);
+    if (org.apache.commons.lang3.StringUtils.isBlank(addressStr)) {
+      String input = request.getReader().lines()
+              .collect(Collectors.joining(System.lineSeparator()));
+      Util.checkBodySize(input);
+      JSONObject jsonObject = JSONObject.parseObject(input);
+      addressStr = jsonObject.getString(addressParam);
+    }
+    if (org.apache.commons.lang3.StringUtils.isNotBlank(addressStr)) {
```
Comment (jiangyy0824) [inline framework/src/main/java/org/tron/core/services/http/Util.java]: Why use "org.apache.commons.lang3.StringUtils" instead of "StringUtils"?

Comment (Parachuteuk) [reply framework/src/main/java/org/tron/core/services/http/Util.java]: it conflicts with StringUtils already used in Util.java

Comment (jiangyy0824) [reply framework/src/main/java/org/tron/core/services/http/Util.java]: you should not use org.apache.commons.lang3.StringUtils directly. Import the org.apache.commons.lang3.StringUtils. And remove the useless import files in "GetBrokerageServlet.java" and "GetRewardServlet.java"

Comment (Parachuteuk) [reply framework/src/main/java/org/tron/core/services/http/Util.java]: Thanks for your advice, will do.

File: framework/src/main/java/org/tron/core/services/http/Util.java [hunk 2]
```diff
@@ -372,7 +375,7 @@ public static String parseMethod(String methodSign, String input) {
     byte[] selector = new byte[4];
     System.arraycopy(Hash.sha3(methodSign.getBytes()), 0, selector, 0, 4);
     //System.out.println(methodSign + ":" + Hex.toHexString(selector));
-    if (StringUtils.isNullOrEmpty(input)) {
+    if (StringUtils.isEmpty(input)) {
```
Comment (jiangyy0824) [inline framework/src/main/java/org/tron/core/services/http/Util.java]: why you use isEmpty instead of isNullOrEmpty

Comment (Parachuteuk) [reply framework/src/main/java/org/tron/core/services/http/Util.java]: they provide the same implementation, isNullOrEmpty  can be replaced with isEmpty, also, no need to import org.pf4j.util.StringUtils

**Pull request merged**